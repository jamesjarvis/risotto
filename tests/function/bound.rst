func a() string {
    return "working"
}

println(a())

func (i int) plus2() int {
    return i + 2
}

func (i int) shout() {
    println("HI I'M "+i.plus2())
    println("HI I'M "+i)
}

30.shout()
