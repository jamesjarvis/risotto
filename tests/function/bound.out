working
HI I'M 32
HI I'M 30
